Formula/w/watchman.rb

```diff
@@ -66,6 +68,10 @@ def install
     bin.install (path/"bin").children
     lib.install (path/"lib").children
     rm_r(path)
+
+    %w[make replicate-subscription wait].each do |file|
+      rewrite_shebang detected_python_shebang, "#{bin}/watchman-#{file}"
+    end
```

Comment:
We can make this simpler:
```suggestion
    rewrite_shebang detected_python_shebang, *bin.children
```